airbyte-integrations/connectors/source-help-scout/source_help_scout/help_scout.yaml

```diff
@@ -0,0 +1,995 @@
+version: 0.1.0
```

Comment:
could you rename this file to `manifest.yaml` and update `source.py` to point to the updated file name? we use that filename convention now